compiler/rustc_erase/Cargo.toml

```diff
@@ -0,0 +1,6 @@
+[package]
+name = "rustc_erase"
```

Comment:
Why is this inside a new crate and not just in `rustc_data_structures`? Creating a crate for 47 lines seems pointless, even if the code is very unsafe and self-contained

Reply:
```rust
// This is a separate crate so that we can `allow(incomplete_features)` for just `generic_const_exprs`
```

Reply:
Ah, didn't see that.

Reply:
Note that `generic_const_exprs` will cause ICEs in other crates than the one using the feature if any of it is used in public APIs